bigquery/datalab-migration/requirements.txt

```diff
@@ -2,4 +2,4 @@ google-cloud-bigquery[pandas,pyarrow]==1.8.1
 datalab==1.1.4
 ipython==7.2.0; python_version > "2.7"
 ipython<=5.5; python_version == "2.7"
-google-cloud-monitoring<=0.28.1
+google-cloud-monitoring==0.28.1
```

Comment:
@tswast any strong feelings about this?

Reply:
I think we had <= because we didn’t want DPEBot to touch it. Looks like Renovate Bot figures out the compatible version for datalab.